packages/discovery-provider/src/api/v1/users.py

```diff
@@ -1813,10 +1816,6 @@ def _get_supportings(self, id: str):
         return success_response(support)
 
 
-full_get_supporting_response = make_full_response(
-    "full_get_supporting", full_ns, fields.Nested(supporting_response_full)
-)
-
 GET_SUPPORTING_ROUTE = "/<string:id>/supporting/<string:supported_user_id>"
```

Comment:
I see we're using the `Supporting` terminology in a few other places. Is it worth cleaning that up?

Reply:
I debated this. The endpoint itself is in use by apiClient. It's the SDK definitions that are wrong.
I don't think "get supporting" is confusing. It was just the pluralized version that was confusing and easy to mess up by forgetting the S. So I would vote we leave the endpoints and singular use case as-is for now to avoid the change being too broad. My main motivation here was to fix the generated SDK methods so I can use them in the client migration.

Reply:
Sounds good to me!